src/default.js

```diff
@@ -609,7 +625,7 @@
           var arg = arguments;
           var rv = orig.apply(this, arg);
           var event;
-          if (typeof Event == "function") {
+          if (isFunction(Event)) {
```

Comment:
O wait, this will fail now of course, because Event can be undefined. 

```suggestion
          if (typeof Event == "function") {
```

Reply:
Ah, there's a naming conflict. There's a variable being defined at line `:693` that's getting in the way of the new helper function. We can rename that variable or remove the new helper function I suppose?

Reply:
Updated, there should now no longer be a naming conflict, and all bits of the code that check if a variable is a function use the new helper method! :)

Reply:
Except that this code should be reverted to `if (typeof Event == "function") {`. Event van be undefined. We could also use `if (isFunction(window.Event)) {`

Reply:
```
> var isFunction = function (func) { return typeof func === "function"; };
> isFunction(undefined);
false
```

I think I don't exactly follow why we can't use the same helper method here, but I'll apply the change as requested!